falcon_kit/mains/ovlp_to_graph.py

```diff
@@ -962,8 +1008,7 @@ def construct_compound_paths(ug, u_edge_data):
 def main(argv=sys.argv):
     import argparse
 
-    parser = argparse.ArgumentParser(description='a example string graph assembler that is desinged for handling diploid genomes',
-            formatter_class=argparse.ArgumentDefaultsHelpFormatter)
```

Comment:
Please leave this line here. Michael Kocher suggested this, and it was a good idea. It shows defaults when you run `program --help`.